sdk/keyvault/Azure.Security.Keyvault.Secrets/src/Azure.Security.Keyvault.Secrets.csproj

```diff
@@ -5,7 +5,7 @@
     <Description>This is the Microsoft Azure Key Vault Secrets client library</Description>
```

Comment:
Why is this project not just including the `$(RequiredTargetFrameworks)` as the target frameworks?

Reply:
lol.. nevermind I see it is but you have both TargetFramework and the TargetFrameworks property in this project. That also explains your CI failure. You should remove the TargetFramework property.

Reply:
Ohh yes! removing!